### PR TITLE
create django project and split settings

### PR DESCRIPTION
**Pull request by OreLana**
# Description
I created the django project and split the settings into dev, production, testing and base settings

# How to Run
- Pull this PR
- Run `virtualenv venv` to create virtual environment
- Run `source venv/bin/activate` to activate virtual environment for mac or ` venv\Scripts\activate.bat` for windows
- Run `pip3 install -r requirements.txt`
- Run `python3 manage.py runserver --settings=hotelms.settings.dev`

### CODE REVIEW

**Review by edemaukabi — COMMENTED** [review]
Looks good!

File: hotelms/settings/production.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from decouple import config
+
+DEBUG = False
+#Production Environment Variables
+DATABASES = {
+  'default': {
+      'ENGINE': 'django.db.backends.postgresql',
+      **dj_database_url.parse(config('DATABASE_URL')),
+  }
+}
```
Comment (eguonoefe) [inline hotelms/settings/production.py:12]: Add a new line here at the end of the file

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SECRET_KEY=somevalue
+USER=somevalue
+PASSWORD=somevalue
```
Comment (eguonoefe) [inline .env.sample:3]: Add a new line here at the end of the file

File: hotelms/settings/base.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    },
+]
+
+WSGI_APPLICATION = 'hotelms.wsgi.application'
+
+
+# # Database
+# # https://docs.djangoproject.com/en/4.0/ref/settings/#databases
+
+# DATABASES = {
```
Comment (eguonoefe) [inline hotelms/settings/base.py:78]: Why is this commented. If it's not being used, it should be deleted not commented out

File: hotelms/settings/testing.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+DATABASES = {
+'default': {
+    'ENGINE': 'django.db.backends.postgresql',
+    'NAME': 'github_actions',
+    'USER': 'postgres',
+    'PASSWORD': 'postgres',
+    'HOST': '127.0.0.1',
+    'PORT': '5432',
+    }
+}
```
Comment (eguonoefe) [inline hotelms/settings/testing.py:14]: Add a new line here at the end of the file

**Review by eguonoefe — CHANGES_REQUESTED** [review]

**Review by eguonoefe — APPROVED** [review]

**Review by eguonoefe — APPROVED** [review]
LGTM

**Pull request merged**